app/models/step_by_step_page.rb

```diff
@@ -63,10 +63,11 @@ def mark_as_published
   end
 
   def mark_as_unpublished
+    now = Time.zone.now
     update(
       published_at: nil,
-      draft_updated_at: nil,
-      status: "draft",
+      draft_updated_at: now,
```

Comment:
I'm not sure why this commit is necessary - could you expand on the commit message a little?

If anything, I'd think the `draft_updated_at` should be left alone, as we have a "Check for broken links" prompt whenever `draft_updated_at` is more recent than the `links_last_checked` date (except for when step by steps have _just_ been scheduled/published).

Reply:
If draft_updated_at is set to nil when we unpublish, it will cause has_draft? to return false but approved_2i requires a step by step to have a draft. Consequently, it will not be possible to unpublish step by step. 

It currently prompts to check for broken links once unpublished. What are your thoughts on adding `|| status.approved_2i?` to `links_checked_since_last_update?` ? 
Would it be ok, not to prompt to check for broken links if the SBS is `approved_2i`? 🤔 

We are showing the prompt when it's a `draft` and `submitted_for_2i` so given that the reviewers check the links manually as well, I would be tempted to assume that the links are correct at that point.

Reply:
Agreed 👍 and thanks for updating the commit message.

Now your first two commit messages have the same headline though 🤔

Reply:
First is Un**scheduled**, second Un**published** 😉

Reply:
D'oh! 🤦‍♂